.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "googletest"]
+	path = googletest
+	url = https://github.com/google/googletest.git
```

Comment:
This generates a bunch of `CMake` stuff and git will keep telling you your working dir is dirty - adding `ignore = dirty` to this configuration will silence them

Reply:
Will change tonight